frontend/src/actions/adminTables.js

```diff
@@ -38,8 +38,8 @@ export const getAdmin = (projectID) => {
                         id: response.data[i].ID,
                         data: response.data[i].Text,
                         metadata: response.data[i].metadata,
-                        similarityPair: response.data[i].similarityPair,
-                        reason: response.data[i].Reason
+                        reason: response.data[i].Reason,
+                        project: projectID
```

Comment:
So I'm getting an odd warning when I adjudicate a card and then go to the skipped tab
<img width="1677" alt="Screen Shot 2022-02-02 at 5 43 04 PM" src="https://user-images.githubusercontent.com/10951569/152250577-ab2af68a-5de5-40ff-9038-10c8878a90f9.png">

Would you mind seeing if you can figure out what it means? It's a warning not an error, but we should probably know what it's mad about.